test/controllers/ApplicationControllerSpec.scala

```diff
@@ -189,10 +190,10 @@ class ApplicationControllerSpec extends BaseSpec with CurrentTaxYear {
       override val allowLowConfidenceSA = false
 
 
-      val r = routeWrapper(buildFakeRequestWithAuth("GET", "/personal-account/do-uplift?redirectUrl=http://example.com")).get
+      val redirection = routeWrapper(buildFakeRequestWithAuth("GET", "/personal-account/do-uplift?redirectUrl=http://example.com")).get
 
-      status(r) shouldBe BAD_REQUEST
-      redirectLocation(r) shouldBe None
+      redirectLocation(redirection) shouldBe None
```

Comment:
does it make more sense to check the status first?

Reply:
Change made :)